src/dune/coq_rules.ml

```diff
@@ -93,15 +93,23 @@ module Context = struct
     ; scope : Scope.t
     ; boot_type : Bootstrap.t
     ; build_dir : Path.Build.t
+    ; backend : Dune_env.Stanza.Coq.backend
     }
 
   let coqc ?stdout_to t args =
     let dir = Path.build (snd t.coqc) in
     Command.run ~dir ?stdout_to (fst t.coqc) args
 
   let coq_flags t =
```

Comment:
Small note, I will move this to `Coq_flags` soon as indeed handling of flags is becoming quite interesting.